doc/user/tutorials/mentalist/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: "MentaLiST: MLST Analysis"
+search_title: "MentaLiST: MLST Analysis"
+description: "A tutorial on how to type data with MentaLiST."
+---
+
+Multi-locus Sequence Typing with MentaLiST
+==========================================
+{:.no_toc}
+
+This is a quick tutorial on how to use IRIDA to analyze data with [MentaLiST][mentalist-github].
+
+* TOC
+{:toc}
+
+Prepare Kmer Database
```

Comment:
This section is more of an Administrator task, so I think it should be moved into a separate section under https://irida.corefacility.ca/documentation/user/administrator/ (https://github.com/phac-nml/irida/blob/development/doc/user/administrator/index.md).  You can then link to it from here.

Reply:
I'm currently linking to [this file](https://github.com/dfornika/irida/blob/2673fd45ec72e7c1da48edbcbd9e5454f6cd43af/doc/administrator/galaxy/pipelines/mentalist/index.md) which covers MentaLiST kmer database installation. Is that ok, or should I add another section under the `user/administrator` page?

Reply:
That looks good to me.  Thanks.